### PR TITLE
ms1051: add new field to Work

### PR DESCRIPTION
**Pull request by MingtaoSun**
@scoen @shuangzhou

### CODE REVIEW

File: src/amberdb/model/Work.java [hunk 1]
```diff
@@ -165,6 +165,12 @@
 
     @Property("form")
     public void setForm(String form);
+    
+    @Property("displayTitlePage")
+    public String getDisplayTitlePage();
```
Comment (scoen) [inline src/amberdb/model/Work.java]: Because this is a flag, perhaps it might be better as a boolean property ?

**Comment by MingtaoSun** [comment]
@scoen, yep, it needs to be a checkbox, so I have changed it to boolean. Thanks.

**Comment by AaronCMuller** [comment]
There is really no reason to update the snapshot version here. Snapshots are meant to be overriden.

**Comment by scoen** [comment]
Aaron is right. 
Let's try not updating the SNAPSHOT version every time.

**Comment by MingtaoSun** [comment]
@AaronCMuller @scoen Fixed

**Pull request merged**